File: client/iih_stub/iih.ci_stub/iih.ci.ord/tmpl/d/CiOrTmplSetItmDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.tmpl.d
{
    /// <summary>
    /// CiOrTmplSetItmDTO 的摘要说明。
    /// </summary>
    public class CiOrTmplSetItmDTO : BaseDTO {

        public CiOrTmplSetItmDTO() {
 
        }

        /// <summary>
        /// 临床医嘱模板套内项目主键标识
        /// </summary>
		public string Id_ciortmplsetitm {
            get { return getAttrVal<string>("Id_ciortmplsetitm",null); }
            set { setAttrVal<string>("Id_ciortmplsetitm", value); }
        }

        /// <summary>
        /// 临床医嘱模板项目
        /// </summary>
		public string Id_ciortmplsrv {
            get { return getAttrVal<string>("Id_ciortmplsrv",null); }
            set { setAttrVal<string>("Id_ciortmplsrv", value); }
        }

        /// <summary>
        /// 套内项目
        /// </summary>
		public string Id_srv {
            get { return getAttrVal<string>("Id_srv",null); }
            set { setAttrVal<string>("Id_srv", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_ciortmplsetitm", "Id_ciortmplsrv", "Id_srv"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.tmpl.d.CiOrTmplSetItmDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/tmpl/d/CiOrTmplSetItmDTO.java
package iih.ci.ord.tmpl.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 临床医嘱模板套内项目 DTO数据 
 * 
 */
public class CiOrTmplSetItmDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 临床医嘱模板套内项目主键标识
	 * @return String
	 */
	public String getId_ciortmplsetitm() {
		return ((String) getAttrVal("Id_ciortmplsetitm"));
	}	
	/**
	 * 临床医嘱模板套内项目主键标识
	 * @param Id_ciortmplsetitm
	 */
	public void setId_ciortmplsetitm(String Id_ciortmplsetitm) {
		setAttrVal("Id_ciortmplsetitm", Id_ciortmplsetitm);
	}
	/**
	 * 临床医嘱模板项目
	 * @return String
	 */
	public String getId_ciortmplsrv() {
		return ((String) getAttrVal("Id_ciortmplsrv"));
	}	
	/**
	 * 临床医嘱模板项目
	 * @param Id_ciortmplsrv
	 */
	public void setId_ciortmplsrv(String Id_ciortmplsrv) {
		setAttrVal("Id_ciortmplsrv", Id_ciortmplsrv);
	}
	/**
	 * 套内项目
	 * @return String
	 */
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}	
	/**
	 * 套内项目
	 * @param Id_srv
	 */
	public void setId_srv(String Id_srv) {
		setAttrVal("Id_srv", Id_srv);
	}
}